package/gluon-offline-ssid/files/etc/config/gluon-offline-ssid

```diff
@@ -0,0 +1 @@
+config settings 'settings'
```

Comment:
Is this OK? I wonder if `main settings` would be better?

original issue here: https://github.com/Freifunk-Nord/gluon-ssid-changer/issues/17

Reply:
You could set it to config offline-ssid 'settings' as e.g. the autoupdater does I think